src/room/InCallView.tsx

```diff
@@ -197,7 +208,20 @@ export function InCallView({
     }
 
     return participants;
-  }, [userMediaFeeds, activeSpeaker, screenshareFeeds, layout]);
+  }, [userMediaFeeds, activeSpeaker, screenshareFeeds]);
+
+  // The maximised participant: either the participant that the user has
+  // manually put in fullscreen, or the focused (active) participant if the
+  // window is too small to show everyone
+  const maximisedParticipant = useMemo(
+    () =>
+      fullscreenParticipant ?? bounds.height <= 500
```

Comment:
I think we ought to check for width too - not sure about the exact use-case but you could have the window with a low height but high width and want to see multiple tiles